man/systemd-detect-virt.xml

```diff
@@ -92,6 +92,11 @@
             <entry>Oracle VM VirtualBox (historically marketed by innotek and Sun Microsystems), for legacy and KVM hypervisor</entry>
           </row>
 
+          <row>
+            <entry><varname>powervm</varname></entry>
+            <entry>IBM PowerVM hypervisor - comes as firmware with some IBM POWER servers</entry>
+          </row>
```

Comment:
hmm, actually the "morerows" field a few rows up needs to be updated from 12 to 13, otherwise the formatted table will be improperly aligned.